services/datamanager/builtin/builtin_test.go

```diff
@@ -603,7 +603,10 @@ func TestSyncDisabled(t *testing.T) {
 	// Make the captureDir where we're logging data for our arm.
 	captureDir := "/tmp/capture"
 	armDir := captureDir + "/arm/arm1/"
+	resetFolder(t, armDir)
+	resetFolder(t, captureDir)
 	defer resetFolder(t, armDir)
+	defer resetFolder(t, captureDir)
```

Comment:
I believe just resetting captureDir will reset all of the subdirectories so this might be all that's needed.

Reply:
Ah missed that it was a subdir of capture, removed the armdir ones